client/google-search-app/src/app/search-form/search-form.component.scss

```diff
@@ -0,0 +1,14 @@
+.parent {
+  overflow: hidden;
+  position: absolute;
```

Comment:
avoid absolute position which is not recommended in next PRs.

Reply:
ok. what is the alternative. @ramsunvtech can you elaborate which one should be used instead?